installer/manifests/keptn/templates/core.yaml

```diff
@@ -108,6 +108,11 @@ spec:
                 secretKeyRef:
                   name: {{ default "keptn-api-token" .Values.apiService.tokenSecretName }}
                   key: keptn-api-token
+            - name: "USERNAME"
+              valueFrom:
+                secretKeyRef:
+                  key:  keptnUsername
+                  name: bridge-credentials
```

Comment:
I don't think you need this here, since the username is used by the bridge directly from the secret.